.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2.1
+
+# Define the jobs we want to run for this project
+jobs:
+  build:
+    docker:
+      - image: pytorch/manylinux-builder:cpu
+    resource_class: 2xlarge+
+    steps:
+      - checkout
+      - run:
+          name: "Checking out PyTorch"
+          command: |   
+            git clone --depth 1 https://github.com/pytorch/pytorch /home/circleci/project/pytorch
+            export PACKAGE_TYPE=manywheel
+            export DESIRED_CUDA=cpu
+            export DESIRED_PYTHON=3.7m
+            export DESIRED_DEVTOOLSET=devtoolset7
+            export DOCKER_IMAGE=pytorch/manylinux-cpu # this may not be needed since we are inside the docker image 
+            export MAX_JOBS=18  # this is ideally a calculated number
```

Comment:
Thanks. The original logic from core strictly upper bounds `MAX_JOBS=18` even when `NUM_CPUS` is greater. Not sure if it's still necessary now but we can apply that here as well: 
```
export MAX_JOBS=${MAX_JOBS:-$(( ${NUM_CPUS} > ${MEMORY_LIMIT_MAX_JOBS} ? ${MEMORY_LIMIT_MAX_JOBS} : ${NUM_CPUS} ))}
```

Reply:
@seemethere alas, it would not work, as `nproc` would return 36 on CircleCI Docker.2xlarge instance, whereas real number of vCPUs available inside of the container is 20.

But perhaps it warrants a comment in .yml file

Reply:
> Minor nit: I would've done `foo` -> $(foo) change in separate pr
Doesn't seem to hurt. Interesting to know why? :)

Reply:
I guess the actual number of CPUs is `Core(s) per socket` not `CPU(s)`:
```
[root@b6fd62bc5078 ~]# lscpu
Architecture:          x86_64
CPU op-mode(s):        32-bit, 64-bit
Byte Order:            Little Endian
CPU(s):                36
On-line CPU(s) list:   0-35
Thread(s) per core:    2
Core(s) per socket:    18
Socket(s):             1
NUMA node(s):          1
Vendor ID:             GenuineIntel
CPU family:            6
Model:                 85
Model name:            Intel(R) Xeon(R) Platinum 8124M CPU @ 3.00GHz
Stepping:              4
CPU MHz:               3400.321
BogoMIPS:              5999.99
Hypervisor vendor:     KVM
Virtualization type:   full
L1d cache:             32K
L1i cache:             32K
L2 cache:              1024K
L3 cache:              25344K
NUMA node0 CPU(s):     0-35
```